tests/integration/components/o-s-s/star-rating-test.ts

```diff
@@ -22,7 +22,23 @@ module('Integration | Component | o-s-s/star-rating', function (hooks) {
                                         @passiveColor={{this.passiveColor}} />`);
       assert.dom('.fas').hasClass(`color-${starColor}`);
     });
+
+    test('Passing @passiveStyle and @passiveColor applies the correct classes to the component', async function (assert) {
```

Comment:
passiveColor is already tested in the loop above.
Either add your `passiveStyle` test to the existing test or move this test out of the `for` loop and test for both `regular` and `solid` styles